src/rpg_database/character_sheet/Background.java

```diff
@@ -57,7 +62,7 @@ public void setSelfInSheet(CharacterSheet characterSheet) {
 		}
 
 		characterSheet.characterData.put(Fields.BACKGROUND, this);
-		characterSheet.characterData.put(Fields.RACE, this.race);
+		characterSheet.characterData.put(Fields.RACE, this.allowedRaces.iterator().next());
```

Comment:
```this.allowedRaces.iterator().next()``` ezt emeld ki légyszi valami privát függvénybe getFirstAllowedRace néven légyszi, VAGY rakd be egy változóba aminek az a neve h firstAllowedRace

Reply:
ok